### PR TITLE
Fix exception that occur after dragging the selection through the image

### PR DESCRIPTION
**Pull request by ma2ciek**
### Suggested merge commit message ([convention](https://github.com/ckeditor/ckeditor5-design/wiki/Git-commit-message-convention))

Fix: Added fallback for `Rect.getDomRangeRects()` if the given range starts in text node. Closes ckeditor/ckeditor5-ui#317.

### CODE REVIEW

File: src/dom/rect.js [hunk 1]
```diff
@@ -361,12 +363,19 @@ export default class Rect {
...
 		// instead and adjust rect's width to simulate the actual geometry of such range.
 		// https://github.com/ckeditor/ckeditor5-utils/issues/153
+		// https://github.com/ckeditor/ckeditor5-ui/issues/317
 		else {
-			const startContainerRect = new Rect( range.startContainer.getBoundingClientRect() );
-			startContainerRect.right = startContainerRect.left;
-			startContainerRect.width = 0;
+			let startContainer = range.startContainer;
+
+			if ( startContainer.nodeType === Node.TEXT_NODE ) {
```
Comment (Reinmar) [inline src/dom/rect.js]: Would be good to have an `isText()` util for that.

Comment (Reinmar) [reply src/dom/rect.js]: Actually, we have it in the DomConverter already :D And we use similar logic in `scroll.js`. So at least `scroll.js` and now `rect.js` should use it. DomConverter not necessarily.

Comment (ma2ciek) [reply src/dom/rect.js]: Can I make the above change as a follow-up?

Comment (Reinmar) [reply src/dom/rect.js]: 👍

**Pull request merged**